README.md

```diff
@@ -19,99 +19,54 @@ You can watch videos of the system being developed on YouTube:
-* /proc filesystem
-* Pseudoterminals (with /dev/pts filesystem)
-* Filesystem notifications
-* CPU and memory profiling
-* VMWare/QEMU mouse integration
-
-## System services
-
-* Launch/session daemon (SystemServer)
-* Compositing window server (WindowServer)
-* Text console manager (TTYServer)
-* DNS client (LookupServer)
-* Network protocols server (RequestServer and WebSocket)
-* Software-mixing sound daemon (AudioServer)
-* Desktop notifications (NotificationServer)
-* HTTP server (WebServer)
-* Telnet server (TelnetServer)
-* DHCP client (DHCPClient)
-
-## Libraries
-
-* C++ templates and containers (AK)
-* Event loop and utilities (LibCore)
-* 2D graphics library (LibGfx)
-* OpenGL 1.x compatible library (LibGL)
-* GUI toolkit (LibGUI)
-* Cross-process communication library (LibIPC)
-* HTML/CSS engine (LibWeb)
-* JavaScript engine (LibJS)
-* Markdown (LibMarkdown)
-* Audio (LibAudio)
-* Digital Signal Processing/Synthesizer Chains (LibDSP)
-* PCI database (LibPCIDB)
-* Terminal emulation (LibVT)
-* Out-of-process network protocol I/O (LibProtocol)
-* Mathematical functions (LibM)
-* ELF file handling (LibELF)
-* POSIX threading (LibPthread)
-* Higher-level threading (LibThreading)
-* Transport Layer Security (LibTLS)
-* HTTP and HTTPS (LibHTTP)
-* IMAP (LibIMAP)
-
-## Userland features
-
-* Unix-like libc and userland
-* Shell with pipes and I/O redirection
-* On-line help system (both terminal and GUI variants)
-* Web browser (Browser)
-* C++ IDE (HackStudio)
-* Desktop synthesizer (Piano)
-* E-mail client (Mail)
-* Various desktop apps & games
-* Color themes
+## Features
+
+* Modern x86 32-bit and 64-bit kernel with pre-emptive multi-threading
+* [Browser](Userland/Applications/Browser/) with JavaScript, WebAssembly, and more (check the spec compliance for [JS](https://libjs.dev/test262/), [CSS](https://css.tobyase.de/), and [WASM](https://libjs.dev/wasm/))
+* Security features (hardware protections, limited userland capabilities, W^X memory, `pledge` & `unveil`, (K)ASLR, OOM-resistance, web-content isolation, state-of-the-art TLS algorithms, ...)
+* [System services](Userland/Services/) (WindowServer, LoginServer, AudioServer, WebServer, RequestServer, CrashServer, ...) and modern IPC
+* Good POSIX compatibility ([LibC](Userland/Libraries/LibC/), Shell, syscalls, signals, pseudoterminals, filesystem notifications, standard Unix [utilities](Userland/Utilities/), ...)
+* POSIX-like virtual file systems (/proc, /dev, /sys, /tmp, ...) and ext2 file system
+* Profiling, debugging and other development tools (Kernel-supported profiling, detailed program analysis with software emulation in UserspaceEmulator, CrashReporter, interactive GUI playground, HexEditor, HackStudio IDE for C++ and more)
+* [Libraries](Userland/Libraries/) for everything from cryptography to OpenGL, audio, JavaScript, GUI, playing chess, ...
+* Support for many common and uncommon file formats (PNG, JPEG, GIF, MP3, WAV, FLAC, ZIP, TAR, PDF, QOI, Gemini, ...)
```

Comment:
Since we already have Gemini here how about a point about the networking stack with different protocols? (it's also not really a file format if I understand correctly)

Reply:
Gemini is also a file format, the native format of the Gemini protocol.